210_Identifying_words/10_Standard_analyzer.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[standard-analyzer]]
-=== standard Analyzer
+=== 标准分析器
 
-The `standard` analyzer is used by default for any full-text `analyzed` string
-field. ((("standard analyzer"))) If we were to reimplement the  `standard` analyzer as a
-<<custom-analyzers,`custom` analyzer>>, it would be defined as follows:
+
+任何全文检索的字符串域都默认使用 `standard` 分析器。((("standard analyzer")))
+如果你想重新实现 `standard` 分析器为 <<custom-analyzers,`custom`  分析器>> ，你可以按照如下所示定义：
```

Comment:
custom 翻译而analyzer 不翻译，后面的 standard 不翻译，而 analyzer 翻译感觉怪怪的。
If we were to reimplement the  `standard` analyzer as a <<custom-analyzers,`custom` analyzer>>, it would be defined as follows:
我记得M大说过附录有词汇对照表，于是干脆全翻译掉，我尝试了下整句重新翻译：
如果我们想要一个 <<custom-analyzers,`自定义` 分析器>>，可以按照如下定义方式重新实现 `标准` 分析器： 
@medcl